### PR TITLE
Remove irrelevant comments

### PR DESCRIPTION
**Pull request by clarencepohh**

### CODE REVIEW

File: src/main/java/data/Event.java [hunk 1]
```diff
@@ -42,17 +42,5 @@ public String getTaskType() {
...
-     * Override function of superclass Task.
-     *
-     * @return The required save format including the task type, status icon, name, start and end date.
-     */
-
-    @Override
-    public String getSaveFormat () {
-        return getTaskType() + " | " + getStatusIcon() + " | " + this.name + " | " + startDate + " | " + endDate;
-    }
 }
```
Comment (Mahesh1772) [inline src/main/java/data/Event.java:45]: Good job removing redundant files

File: src/main/java/data/Task.java [hunk 2]
```diff
@@ -76,14 +76,4 @@ public String getStatusIcon() {
...
-
-    /**
-     * Returns the String save format used to save Hachi task data.
-     *
-     * @return ? which represents an unclassified task.
-     */
-
-    public String getSaveFormat () {
-        return "?";
-    }
```
Comment (Mahesh1772) [inline src/main/java/data/Task.java:88]: Thanks for cleaning up the code!

**Review by Mahesh1772 — COMMENTED** [review]
LGTM!

**Review by NicholasTanYY — COMMENTED** [review]
LGTM. Good job on removing redundant methods and comments.

**Pull request merged**